clients/filesystem-fuse/Makefile

```diff
@@ -62,6 +62,9 @@ doc-test:
 unit-test: doc-test
 	cargo test --no-fail-fast --lib --all-features --workspace
 
+test-it:
```

Comment:
It's really hard to distinguish `test-it` and `test-fuse-it`, please use a better name or add more comment